main/tests/server/src/com/google/refine/tests/expr/functions/strings/ToFromConversionTests.java

```diff
@@ -138,7 +138,7 @@ public void testToDate() throws CalendarParserException {
       Assert.assertTrue(invoke("toDate", (Object) null) instanceof EvalError);
       Assert.assertTrue(invoke("toDate", "") instanceof EvalError);
       Assert.assertTrue(invoke("toDate", 1.0) instanceof EvalError);
-      Assert.assertTrue(invoke("toDate", "2012-03-01","xxx") instanceof EvalError); // bad format string
+      //Assert.assertTrue(invoke("toDate", "2012-03-01","xxx") instanceof EvalError); // bad format string
```

Comment:
Disabling tests or changing expected behavior shouldn't be done as part of a refactoring. The whole point of refactoring is to keep the same behavior.

I've reenabled this test and corrected the code to return an error here.

Reply:
@tfmorris That's very true.  But to @ostephens credit, he was doing his best here and was leaning on senior developers to help spot things for him.  He has admitted in the past that even now he's still learning things.
Again, we are so grateful to have you around again!